util/travis-deploy-staging.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Helper script for posting a GitHub comment pointing to the deployed environment,
+# from Travis CI. Also see deploy.sh
+
+APP_PATH=$1
+
+UTIL_DIR="$(dirname $0)"
+source "${UTIL_DIR}/logging.sh"
+
+if [ ${TRAVIS_SECURE_ENV_VARS} == false ]; then
```

Comment:
nit: double-square-brackets preferred, because [reasons](https://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash). That said, I think we have more single than double at this point in this project ¯\_(ツ)_/¯